opengrok-web/src/main/java/org/opengrok/web/api/constraints/PositiveDuration.java

```diff
@@ -37,7 +37,7 @@
 @Retention(RetentionPolicy.RUNTIME)
 public @interface PositiveDuration {
 
-    String message() default "{org.opengrok.indexer.web.api.constraints.PositiveDuration.message}";
+    String message() default "{org.opengrok.web.api.constraints.PositiveDuration.message}";
```

Comment:
I'm not familiar with this, can you in short explain what it does?

Reply:
https://www.baeldung.com/javax-validation

The default message if the validation of the bean does not pass.

Reply:
thanks